v3/src/models/document/document-content.ts

```diff
@@ -184,6 +184,19 @@ export const DocumentContentModel = BaseDocumentContentModel
           return self.createTile(tileType, options)
         }
       }
+    },
+    // Hide the tile if it's a singleton and can be hidden. Otherwise, delete it.
+    deleteOrHideTile(tileId: string) {
+      const tile = self.getTile(tileId)
+      const tileInfo = getTileContentInfo(tile?.content.type)
+      if (tileInfo?.isSingleton) {
```

Comment:
I think we need another property here because the case table and data card tiles should also hide, but they're not technically singletons. I'm thinking there should be a `hideOnClose` property on `TileContentInfo` and then this would be
```suggestion
      if (tileInfo?.isSingleton || tileInfo?.hideOnClose) {
```

Reply:
Good call 👍  I added the `hideOnClose` property and set it to true for the table and data card. I thought about going through the code and checking this flag whenever `toggleNonDestroyableTileVisibility()` is called, but that seemed outside the scope of this story.

This generally works, but it does seem to fail when the table is part of the loaded document. For example, deleting the table in mammals will hide it, but then showing the table will create a new one rather than showing the hidden one. If you then "delete" the new one and show the table, it will correctly show the 2nd table. I haven't dug into why the first table is being ignored after it has been hidden. Do you think I should do that?

Reply:
>I haven't dug into why the first table is being ignored after it has been hidden. Do you think I should do that?

I think it's worth a quick look because it suggests there's something we don't understand here that might affect the implementation. Once we know what's going on, I suspect the fix will be straightforward.

Reply:
Alright, looking into this more, I think the issue is that I was using an older document I had saved a couple of weeks ago. It looks like support for showing hidden tables/cards wasn't added until about half a week ago, so my document was just lacking the infrastructure needed to associate my "deleted" table with the dataset when I was showing the table again. Using a fresh document, and even saving and loading it again, results in the correct behavior.